Makefile

```diff
@@ -84,6 +84,7 @@ $(LIBCLANG_PATH): $(LIBCLANG_FILES)
 else ifeq ($(shell uname -s),Darwin)
     MD5SUM = md5
     LIBCLANG_PATH = $(abspath $(LLVM_BUILDDIR))/lib/libclang.a
+    CGO_CXXFLAGS=-I/Library/Developer/CommandLineTools/usr/include/c++/v1
```

Comment:
This is a platform-specific change that really doesn't belong in this PR. And honestly I feel like this patches over some existing issue on your system: it seems to work fine for most other people.

Reply:
Yeah, you are right.